CONTRIBUTING.md

```diff
@@ -12,9 +12,15 @@ Shareon welcomes contributions and corrections. Before contributing, please make
    ```
 
 4. Create a new branch from the latest commit on `main`
-5. Start hacking on the new branch
-6. Commit and push to the new branch
-7. Make a pull request to `main`
+5. Install [pnpm](https://pnpm.io/):
```

Comment:
Let's just put it into step 0 and let others figure out how they would want to install it